resources/views/participants/show.blade.php

```diff
@@ -193,8 +185,14 @@
 		@if ($viewType == 'profile')
 		<p>De opgegeven informatie per vak kunt u bekijken door op 'vakken bewerken' te klikken.</p>
 		@endif
+
+		<div class="row">
+			@include('partials.comments', [ 'comments' => $participant->comments, 'type' => 'App\Participant', 'key' => $participant->id ])
+		</div>
```

Comment:
Als je de `div class='row'` wrapper weghaalt, is de uitlijning met de andere zaken in de kolom veel mooier.